ir/expression.def

```diff
@@ -482,6 +482,7 @@ class StructExpression : Expression {
         size_t size = components.size();
         return components.at(size - 1)->is<IR::NamedDots>();
     }
+    cstring toString() const;
```

Comment:
This should either be marked `override` or implemented in the form of 
```C++
toString {
    cstring str = "{";
    if (!components.empty()) {
        cstring exprStr = components.at(0)->expression->toString();
        str += " " + components.at(0)->toString() + " = " + exprStr;
    }
    for (unsigned i = 1; i < size(); i++) {
        cstring exprStr = components.at(i)->expression->toString();
        str += ", " + components.at(i)->toString() + " = " + exprStr;
    }
    return str + " }";
}
```
like with other nodes.